cranelift/zkasm_data/benchmarks/fibonacci/state.csv

```diff
@@ -0,0 +1,3 @@
+Test,Status,Cycles
+from_rust,compilation failed,
+handwritten_wat,pass,190024
```

Comment:
LGTM. Just interesting, what is ratio from cycles to seconds?

Reply:
Yeah, that would be very interesting to know. I'll be working on figuring this out next.
We basically know that ZK ASM is ~10KHz machine, so now we also need to benchmark that Fibonacci code on normal processor and compute the ratio.